cat-tinder/connecting/cat-form.md

```diff
@@ -17,111 +17,129 @@ The `createCat()` fetch call is going to be a little bit different than `getCats
 
```

Comment:
this section - which is outside the diff - hasn't been updated to be accurate with the new information at all. I needs to cover the new function instead of the old one